src/Fabulous/ViewNode.fs

```diff
@@ -3,13 +3,78 @@
 open Fabulous
 
 /// Define the logic to apply diffs and store event handlers of its target control
+[<Sealed>]
 type ViewNode(parentNode: IViewNode voption, treeContext: ViewTreeContext, targetRef: System.WeakReference) =
 
     // TODO consider combine handlers mapMsg and property bag
     // also we can probably use just Dictionary instead of Map because
     // ViewNode is supposed to be mutable, stateful and persistent object
     let mutable _handlers: Map<AttributeKey, obj> = Map.empty
 
+    member inline private this.ApplyScalarDiffs(diffs: ScalarChanges inref) =
```

Comment:
Did you inline it to remove the method call cost?

Reply:
yeah, I was thinking to literally inline it into one function, but decided to preserve the structure while still inlining it for perf reasons